e2e/rbd.go

```diff
@@ -421,6 +421,73 @@ var _ = Describe("RBD", func() {
 				validateRBDImageCount(f, 0, defaultRBDPool)
 			})
 
+			By("create an erasure coded PVC and bind it to an app", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass: %v", err)
+				}
+				err = createRBDStorageClass(
+					f.ClientSet,
+					f,
+					defaultSCName,
+					nil,
+					map[string]string{
+						"dataPool": "ec-pool",
+						"pool":     "replicapool",
+					},
+					deletePolicy)
+				if err != nil {
+					e2elog.Failf("failed to create storageclass with error %v", err)
+				}
+				pvc, app, err := createPVCAndAppBinding(pvcPath, appPath, f, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to create pvc and application binding: %v", err)
+				}
+				err = checkPVCDataPoolForImageInPool(f, pvc, defaultRBDPool, "ec-pool")
```

Comment:
validateRBDImageCount

Reply:
Yes, it is validated after the deletion of created PVC, to make sure no stale backend images are left.

Reply:
using `"ec-pool"`, and not the `erasureCodedPool` variable here, please correct that.